FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -34,6 +34,8 @@ const auctionBlob = navigator.getInterestGroupAdAuctionData({
     },
     'https://buyer2.origin.example.com': {}
   }
+const request = auctionBlob.request;
+const requestId = auctionBlob.requestId;
 });
```

Comment:
erm wait the new variables should be outside the function call

Reply:
Oops. fixed.